### ISSUE TITLE
0.31 crashes on Android 8 and 9

### ISSUE THREAD

**Issue by skitt** [meikpiep/gauguin#232]
I know, 8 and 9 are obsolete, so feel free to ignore this (but I have a fix, I’ll open a PR).

On Android 8 and 9, Gauguin 0.31 crashes during initialisation:

```
10-06 12:40:30.344 10495 26340 26340 E AndroidRuntime: Caused by: java.lang.IllegalStateException: Font variation settings are already set.
10-06 12:40:30.344 10495 26340 26340 E AndroidRuntime:  at android.graphics.Typeface$Builder.setFontVariationSettings(Typeface.java:471)
10-06 12:40:30.344 10495 26340 26340 E AndroidRuntime:  at org.piepmeyer.gauguin.ui.grid.GridFontHolder.<init>(GridFontHolder.kt:20)
10-06 12:40:30.344 10495 26340 26340 E AndroidRuntime:  at org.piepmeyer.gauguin.ui.grid.GridPaintHolder.<init>(GridPaintHolder.kt:49)
10-06 12:40:30.344 10495 26340 26340 E AndroidRuntime:  at org.piepmeyer.gauguin.ui.grid.GridUI.<init>(GridUI.kt:48)
10-06 12:40:30.344 10495 26340 26340 E AndroidRuntime:  ... 31 more
```

**Comment by meikpiep** [comment]
Thanks, @skitt ! I will have a look tommorrow at the PR - much appreciated.

**Comment by meikpiep** [comment]
I will do a release with this fix asap.

**Comment by meikpiep** [comment]
Just released version 0.31.1 https://github.com/meikpiep/gauguin/releases/tag/v0.31.1 containing this fix.

This will be available in 1-2 days via Google play and 2-4 days via F-Droid.

**Comment by skitt** [comment]
Thanks for the quick release! I meant to confirm a month ago that this did indeed fix the issue.
